modules/social_features/social_tagging/social_tagging.module

```diff
@@ -211,12 +221,12 @@ function _social_tagging_node_form_validate($form, FormStateInterface $form_stat
 /**
  * Helper function to get the current default tagging values of a node.
  */
-function _social_tagging_node_form_defaults_values(Node $node) {
+function _social_tagging_node_form_defaults_values(Entity $entity) {
   // Init.
   $default_value = [];
   // If the node exists, we need to get the default value.
-  if ($node instanceof Node && $node->id() !== NULL) {
-    foreach ($node->get('social_tagging')->getValue() as $key => $value) {
+  if ($entity instanceof Entity && $entity->id() !== NULL) {
```

Comment:
We should be using `EntityInterface` here instead.

This allows compatibility with classes that don't inherit Entity but do implement the methods we require.